scripts/grafana-server/custom.ini

```diff
@@ -1,5 +1,11 @@
+app_mode = test
+
+[security]
+content_security_policy = true
+content_security_policy_template = """require-trusted-types-for 'script'; script-src 'self' 'unsafe-eval' 'unsafe-inline' 'strict-dynamic' $NONCE;object-src 'none';font-src 'self';style-src 'self' 'unsafe-inline' blob:;img-src * data:;base-uri 'self';connect-src 'self' grafana.com ws://$ROOT_PATH wss://$ROOT_PATH;manifest-src 'self';media-src 'none';form-action 'self';"""
```

Comment:
You're right, we can probably update this in default.ini. The problem at the moment is that it will start throwing errors if you don't have the feature flag enabled. So maybe we can wait until we have removed the feature flag.

Reply:
idea!  rather than a feature toggle and/or future explicit config, can we trigger the use of trusted types based on if the CSP includes "require-trusted-types" 🤔 

I think this would be safe to include now -- since any CSP that requires trusted types would fail

Reply:
@KristianGrafana -- what about something like:
```
forceTrustedTypes = strings.Contains( content_security_policy_template, "require-trusted-types") && !content_security_policy_report_only
```
This way the frontend will not sanitize when in report only mode?

Reply:
@ryantxu Something like that could work. Had a chat with @tskarhed and had an idea that we could create a more verbose output if TT is enabled in report only mode using `console.log` - this way we still have a policy defined but the output will be different if we run CSP in enforced or report only mode, and no feature flag is required, only the CSP directive.